src/test/java/com/GeorgesServer/app/FormHandlerTest.java

```diff
@@ -3,58 +3,58 @@
 import com.GeorgesServer.app.com.GeorgesServer.handler.FormHandler;
 import com.GeorgesServer.app.com.GeorgesServer.request.ClientRequest;
 import com.GeorgesServer.app.com.GeorgesServer.response.HttpResponseBuilder;
+import com.GeorgesServer.app.com.GeorgesServer.response.ServerResponse;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 class FormHandlerTest {
-    private FormHandler formHandler;
+    private FormHandler subject;
     private ClientRequest mockClientRequest;
-    private HttpResponseBuilder mockResponseBuilder;
+    private String expectedCode = "200";
+    private String expectedMsg = "OK";
+    private String expectedVersion = "HTTP/1.1";
 
     @BeforeEach
     public void setUp() {
         mockClientRequest = mock(ClientRequest.class);
-        mockResponseBuilder = mock(HttpResponseBuilder.class);
-        formHandler = new FormHandler(mockResponseBuilder);
+        subject = new FormHandler();
     }
 
     @Test
     void formHandlerCallsTheCorrectMethodsWhenGivenAGETRequest() {
-
         when(mockClientRequest.getMethod()).thenReturn("GET");
 
-        formHandler.handle(mockClientRequest);
+        ServerResponse result = subject.handle(mockClientRequest);
 
-        verify(mockResponseBuilder).buildHttpVersion();
-        verify(mockResponseBuilder).buildOkStatus();
-        verify(mockResponseBuilder).getResponse();
+        assertEquals(expectedCode, result.getStatusCode());
+        assertEquals(expectedMsg, result.getStatusMsg());
+        assertEquals(expectedVersion, result.getHttpVersion());
     }
 
     @Test
     void formHandlerCallsTheCorrectMethodsWhenGivenAPOSTRequest() {
```

Comment:
post requests typically have a body, so I might add a test for that.

Reply:
Will do, thanks.